stock_picking_consolidation_priority/tests/test_consolidation_priority.py

```diff
@@ -298,6 +298,16 @@ def assert_priority(self, changed_moves, unchanged_moves):
                 ),
             ),
         )
+        changed_pickings = changed_moves.picking_id
+        unchanged_pickings = unchanged_moves.picking_id
+        changed_ok = all(
+            picking.priority == expected_priority for picking in changed_pickings
+        )
+        unchanged_ok = all(
+            picking.priority == PROCUREMENT_PRIORITIES[0][0]
+            for picking in unchanged_pickings
+        )
+        self.assertTrue(changed_ok, unchanged_ok)
```

Comment:
```suggestion
        self.assertEquals(changed_ok, unchanged_ok)
```

Reply:
I did not quiet take your suggestion, but that was a good catch, thank you.